docs/setup/installation.mdx

```diff
@@ -29,8 +29,38 @@ Before proceeding with the installation, ensure that you are targeting the corre
     ```
   </Tab>
   <Tab title="GitHub Releases">
-    Download the latest release from the [GitHub Releases](https://github.com/odigos-io/odigos/releases)
-    page and extract the binary to a directory in your `PATH`.
+  1. Download the Latest Version: Head over to the [Odigos GitHub Releases](https://github.com/odigos-io/odigos/releases) page and click on the latest version available. You’ll see a list of files—choose the one that matches your operating system (Windows, macOS, or Linux).
+
+  2. Extract the Downloaded File: Once the file is downloaded, it will likely be in a compressed format like `.zip` or `.tar.gz`. To use the program, you’ll first need to extract (or unzip) it. Right-click on the file and select "Extract Here" or use a similar option, depending on your operating system.
+
+  3. Move the File to a Folder in Your `PATH`. You’ll need to move this file to a folder where your computer looks for programs, known as your "PATH."
+
+  4. For Windows: Move the extracted file to a folder like `C:\Windows\System32` or any folder listed in your system’s PATH. You can find out where by typing echo `%PATH%` in Command Prompt.
+
+  5. For macOS/Linux: Move the file to a folder like `/usr/local/bin/` or `/usr/bin/`, both of which are common folders included in the PATH. You can check your PATH by opening the terminal and typing echo $PATH.
```

Comment:
nit: echo $PATH should be with back ticks

Reply:
and fixed that too